clients/analysis/src/model/NodeDTO.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2025 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+import { Location } from "./cfast";
+import { Node } from "./Node";
+
+export class NodeDTO {
```

Comment:
Half of this stuff is UI related and should not be here. The type defined in `./Node` seems like the correct one.